zjit/src/codegen.rs

```diff
@@ -170,11 +170,10 @@ fn gen_iseq(cb: &mut CodeBlock, iseq: IseqPtr) -> Option<(CodePtr, Vec<(Rc<Branc
     }
 
     // Convert ISEQ into High-level IR
```

Comment:
```suggestion
    // Convert ISEQ into High-level IR and optimize HIR
```

Reply:
This comment is otherwise misleading

Reply:
👌 https://github.com/ruby/ruby/pull/13196/commits/8f56d47dd98815ff010c8c336e061345f46bfc6a